src/main/java/doit/apihw/api/service/AuthService.java

```diff
@@ -52,11 +52,15 @@ public AuthLoginResponse signUp(AuthSignUpRequest request) {
     // TODO : 로그인을 진행한다.
     public AuthLoginResponse login(AuthLoginRequest request) {
         // DB에서 memberLoginId와 memberPassword를 조회하여 일치하는 회원이 있는지 확인한다.
-
+        Member member = memberRepository.findByMemberLoginIdAndMemberPassword
+                (request.getMemberLoginId(), request.getMemberPassword());
         // 만약 일치하는 회원이 없다면, IllegalArgumentException을 발생시킨다.
-
+        if(member == null) {
+            throw new IllegalArgumentException("존재하지 않는 회원입니다. 다시 시도해주세요.");
+        }
         // 일치하는 회원이 있다면, MemberResponse로 변환하여 반환한다.
-        return null;
+        else
+            return AuthLoginResponse.from(member);
     }
```

Comment:
깔끔하게 잘 작성해주셨습니다!
한 가지 확장성 측면에서 고려해보자면, 다음과 같이 작성해보아도 좋을 것 같아요

```suggestion
        // 만약 일치하는 회원이 없다면, IllegalArgumentException을 발생시킨다.
        if(member == null) {
            throw new IllegalArgumentException("존재하지 않는 회원입니다. 다시 시도해주세요.");
        }
        
        // ( 가정 ) 만약 회원이 차단된 회원이라면, IllegalArgumentException을 발생시킨다.
        if(member.isBanned()) {
            throw new IllegalArgumentException("차단된 회원입니다. 고객 문의로 연락주세요");
        }

        
        // 일치하는 회원이 있다면, MemberResponse로 변환하여 반환한다.
        return AuthLoginResponse.from(member);
    }
```
해당 내용을 다루는 좋은 글 공유드리곘습니다.
https://limdingdong.tistory.com/8